examples/demo/basic/minard_napoleon.py

```diff
@@ -0,0 +1,243 @@
+    (32.0, 54.8, 30.4, 54.4, 24000, -1, 1),
+    (30.4, 54.4, 29.2, 54.4, 20000, -1, 1),
+    (29.2, 54.4, 28.5, 54.3, 50000, -1, 1),
+    (28.5, 54.3, 28.3, 54.3, 50000, -1, 1),
+    (28.3, 54.3, 27.5, 54.5, 48000, -1, 1),
+    (27.5, 54.5, 26.8, 54.3, 40000, -1, 1),
+    (26.8, 54.3, 26.4, 54.4, 12000, -1, 1),
+    (26.4, 54.4, 24.6, 54.5, 14000, -1, 1),
+    (24.6, 54.5, 24.4, 54.4, 8000, -1, 1),
+    (24.4, 54.4, 24.2, 54.4, 4000, -1, 1),
+    (24.2, 54.4, 24.1, 54.4, 10000, -1, 1),
+
+    (28.7, 55.5, 29.2, 54.4, 30000, -1, 2),
+
+    (24.6, 55.8, 24.2, 54.4, 6000, -1, 3),
+],
+    dtype=[
+        ('start_lon', float), ('start_lat', float), ('end_lon', float),
+        ('end_lat', float), ('size', float), ('direction', float),
+        ('group', float)
+    ],
+)
+
+
+def _create_plot_component():
+    army_lat = np.column_stack([army['start_lat'], army['end_lat']]).reshape(-1)
+    army_lon = np.column_stack([army['start_lon'], army['end_lon']]).reshape(-1)
+
+    plot_data = ArrayPlotData(
+        army_lon=army_lon,
+        army_lat=army_lat,
+        army_size=army['size'],
+        army_color=army['direction']*army["group"],
+        towns_lat=towns['lat'],
+        towns_lon=towns['lon'],
+        towns=towns['town'],
+        temp_lon=temperatures['lon'],
+        temp=temperatures['temp'],
+        temp_date=temperatures['date'],
+    )
+
+    map_plot = Plot(plot_data)
+    map_plot.x_grid = None
+    map_plot.y_grid = None
+    map_plot.x_axis.orientation = 'top'
+    map_plot.x_axis.title = 'Longitude'
+    map_plot.y_axis.title = 'Latitude'
+    map_plot.title = "Minard's Map of Napoleon's Russian Campaign"
+    map_plot._title.overlay_position = "inside top"
+    map_plot._title.hjustify = "left"
+    map_plot._title.vjustify = "bottom"
+    map_plot.plot(
+        ("army_lon", "army_lat", "army_color", "army_size"),
+        type="cmap_segment",
+        name="my_plot",
+        color_mapper=viridis,
+        border_visible=True,
+        bgcolor="white",
+        size_min=1.0,
+        size_max=128.0,
+    )
+    map_plot.plot(
+        ("towns_lon", "towns_lat"),
+        type="scatter",
+    )
+    map_plot.plot(
+        ("towns_lon", "towns_lat", "towns"),
+        type="text",
+        text_margin=4,
+        h_position='right',
+        text_offset=(4, 0),
+    )
+    map_plot.plot_1d(
+        ('temp_lon'),
+        type='line_scatter_1d',
+        alpha=0.5,
+        line_style='dot',
+    )
+    map_plot.index_range.high_setting = 38
+    map_plot.index_range.low_setting = 23
+    map_plot.value_range.high_setting = 56.0
+    map_plot.value_range.low_setting = 53.5
+    map_plot.tools.extend([
+        PanTool(map_plot),
+        ZoomTool(map_plot),
+    ])
+
+    temp_plot = Plot(plot_data, height=100)
+    temp_plot.index_range = map_plot.index_range
+    temp_plot.x_grid = None
+    temp_plot.x_axis = None
+    temp_plot.y_axis.orientation = 'right'
+    temp_plot.y_axis.title = u'Temp (°C)'
```

Comment:
I was looking at this on Wikipedia (https://en.wikipedia.org/wiki/French_invasion_of_Russia#/media/File:Minard.png), and it looks like the temperature may be in degrees Reaumur.